Checkout/Checkout/Source/Model/Address.swift

```diff
@@ -7,6 +7,7 @@
 
 import Foundation
 
+/// Initializes a Address object.
```

Comment:
@patrick-hoban-cko please review the docs changes here.

Reply:
Quite hard to read through and check everything, but for the first 20 or so files it looks good @deepesh-vasthimal-cko

Reply:
@patrick-hoban-cko you don't need to review all the doc changes they are pretty much standard, If you can just have a look at all the swift files which has 3 slashes comments /// you can review those strings.